src/main/java/teammates/ui/controller/InstructorCourseDetailsPageAction.java

```diff
@@ -49,7 +49,7 @@ public ActionResult execute() throws EntityDoesNotExistException {
         List<StudentAttributes> students = logic.getStudentsForCourse(courseId);
```

Comment:
If I'm not wrong, I believe you could omit this too. We can check if the course does not have any students (line 54) using `courseDetails` (line 47)